ooniprobe/Info.plist

```diff
@@ -11,11 +11,11 @@
 	<key>CFBundleInfoDictionaryVersion</key>
 	<string>6.0</string>
 	<key>CFBundleName</key>
-	<string>$(PRODUCT_NAME)</string>
```

Comment:
Isn't it possible to just change the `PRODUCT_NAME` variable, instead of hardcoding `OONI Probe` in here?

Reply:
I should check if it's possible without renaming every single folder called 'ooniprobe' in the project. I found this answer on stackoverflow

Reply:
Ok it's fine to change PRODUCT_NAME, however it does some changes in the project settings like
 -           BuildableName = "ooniprobe.app"
 +           BuildableName = "OONI Probe.app"

should I proceed?

Reply:
Yes